internal/store/multitenant_database.go

```diff
@@ -137,6 +143,7 @@ func (sqlStore *SQLStore) UpdateMultitenantDatabase(multitenantDatabase *model.M
 			"RawInstallationIDs": multitenantDatabase.RawInstallationIDs,
 		}).
 		Where(sq.Eq{"ID": multitenantDatabase.ID}).
+		Where(sq.Eq{"VpcID": multitenantDatabase.VpcID}).
```

Comment:
We don't really need this, right? I actually don't think we need any `Where`s for inserting a new row. They must be ignored.

Reply:
This is not inserting a new row, it's updating existing one. I think the VPC ID is redundant but  LockAcquiredBy I think is important redundancy to have given that we use this table primarily for locking things. We want to make sure that only locked resources get updated. ID is definitely needed.